cmd/deployment/show.go

```diff
@@ -32,7 +35,10 @@ const showExample = `
   ecctl deployment show <deployment-id> --kind kibana
 
 * Shows apm resource information from a given deployment with a specified ref-id.
-  ecctl deployment show <deployment-id> --kind apm --ref-id apm-server`
+  ecctl deployment show <deployment-id> --kind apm --ref-id apm-server
+
+* Return the current deployment state as a valid create / update payload.
```

Comment:
```suggestion
* Return the current deployment state as a valid update payload.
```

I don't think this would work for create, would it? It has a `prune_orphans` field

Reply:
That's right!